app/styles/ember-rdfa-editor/_c-annotation.scss

```diff
@@ -62,8 +62,6 @@ $say-annotation-background-color: var(--au-gray-100) !default;
   [typeof]:not([typeof='foaf:Document']),
   [data-type],
   .mark-highlight-manual {
-    border-bottom: 0.2rem dotted var(--au-gray-300);
```

Comment:
What is the reason for the style changes?

Reply:
The dotted line is no longer necessary to show what is contained within rdfa elements, as we now have the boxes to show this. Having both looks very broken. I guess maybe we should toggle whether this is displayed based on whether we're using the rdfa-aware editor or not?

Reply:
It should also be possible to control these styles using the `rdfa-annotations` class. Which you can control using `editorOptions.showRdfa`. I suspect we will also remove a lot of the css of these rdfa-annotations, as the blocks we have now are IMO a better solution.

Reply:
I would drop the style change from this PR, we can still remove it later and you can turn it off by setting `editorOptions.showRdfa` to `false`.

Reply:
I've reverted the css change and removed `showRdfa` from the editable nodes route.